introduction.md

```diff
@@ -0,0 +1 @@
+#I'm Vyzaldy Sanchez, and I'm practicing Mastering GITHUB :D
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.